sources/web/datalab/polymer/modules/base-api-manager/base-api-manager.ts

```diff
@@ -98,14 +98,22 @@ abstract class BaseApiManager implements ApiManager {
 
   public abstract getBasePath(): Promise<string>;
 
-  public async sendRequestAsync(url: string, options?: XhrOptions): Promise<string> {
+  public async sendRequestAsync(url: string, options?: XhrOptions, prependBasepath = true)
+      : Promise<any> {
     const basepath = await this.getBasePath();
-    return this._xhrJsonAsync(basepath + url, options);
+    if (prependBasepath) {
```

Comment:
I think the `const basepath = ...` line should go inside this branch rather than before it.

Reply:
Done.